tests/redisearch.ts

```diff
@@ -28,8 +28,19 @@ describe('RediSearch Module testing', async function() {
         });
         await client.connect();
         await redis.connect();
+        try {
+            await client.redis.flushall();
+            await client.dropindex(index);
+            await client.dropindex(`${index}1`);    
+            await client.dropindex(`${index}-searchtest`);    
+            await client.dropindex(`${index}-droptest`);    
```

Comment:
Why are you dropping indexes? if this was done personally for your runs please remove what ever is no longer relevant for the actual unit tests.

Reply:
if indexes do not exist they throw an error.

Reply:
they are relevant actually. Every time test should start with an empty Redis instance. How do you run your tests?

Reply:
I actually run these tests via CI only, it's run on PR commit